fiftyone/core/stages.py

```diff
@@ -3206,9 +3217,10 @@ def _make_labels_pipeline(self, sample_collection):
         # our intention is not to remove other fields from the schema, only to
         # empty their sample fields in the returned view
         #
-        stage = SelectFields(list(self._labels_map.keys()))
-        stage.validate(sample_collection)
-        pipeline.extend(stage.to_mongo(sample_collection))
+        if self._select_fields:
```

Comment:
Per our online discussion- I think this parameter should be removed. The serialized `SelectFields()` stage here shouldn't affect the schema of the dataset, its just here to omit *all* labels from fields that don't appear in `_labels_map`, which is what the user wants (otherwise they would have included those fields in their selection).

It occurs to me that this might have been added in the spirit of helping the extended view get access to *all* data in case the user wants to open a sample in the expanded modal and view all data. But, I think the workaround/hack for that might be elsewhere, not here.